src/Installers/Windows/AspNetCoreModule-Setup/ANCMV2/AncmV2.wixproj

```diff
@@ -81,4 +81,8 @@
                       '$(PackageIconFullPath)' ^
                       '$(PackageLicenseExpression)' " />
   </Target>
+  
+  <Target Name="BeforeBuild" Condition="'$(Platform)' == 'arm64'">
```

Comment:
@halter73 The forwarder dll files are generated by this target. You might review MSBuild binlog to see why it wasn't triggered when you built the artifacts. I guess it was caused by condition mismatch.

Reply:
Could it be that this `BeforeBuild` target is simply missing from `AncmIISExpressV2.wixproj`? I tried pushing to the `lextm-47290` branch at `git@github.com:lextudio/httpplatformhandlerv2.git` to update this PR and test it, but I got a permission error.

Reply:
En, I didn't add this target to `AncmIISExpressV2.wixproj` because not necessary if building everything out. But like you found out, if just building a few projects (not the entire solution) then that can be a problem.

I just pushed a new commit as you suggested to address this.